### PR TITLE
Match prometheus export metrics format

### PR DESCRIPTION
**Pull request by jtama**
Links #30576

### CODE REVIEW

**Review by ebullient — APPROVED** [review]

File: docs/src/main/asciidoc/telemetry-micrometer-tutorial.adoc [hunk 1]
```diff
@@ -77,16 +77,21 @@ Dimensional labels are added for the request uri, the HTTP method
...
+http_server_requests_seconds_max{method="GET",outcome="SUCCESS",status="200",uri="/example/prime/{number}"} 0.017385896
 #
 ----
 
 NOTE: Metrics appear lazily, you often won't see any data for your endpoint until it is accessed.
 
+.Exported metrics format
+
+By default, the  metrics are exported using micrometer `application/openmetrics-text`, you can revert to the former
+format by specifying the `Accept` header request to `plain/text`.
```
Comment (gsmet) [inline docs/src/main/asciidoc/telemetry-micrometer-tutorial.adoc:93]:
```suggestion
By default, the  metrics are exported using the Micrometer format `application/openmetrics-text`,
you can revert to the former format by specifying the `Accept` request header to `plain/text`.
```

File: docs/src/main/asciidoc/micrometer.adoc [hunk 2]
```diff
@@ -398,6 +398,11 @@ string. For example, setting
 `quarkus.micrometer.binder.http-server.match-patterns=/example/prime/[0-9]+=/example/{jellybeans}` would use the value
 `/example/{jellybeans}` for the uri attribute any time the requested uri matches `/example/prime/[0-9]+`.
 
+.Exported metrics format
+
+By default, the  metrics are exported using micrometer `application/openmetrics-text`, you can revert to the former
+format by specifying the `Accept` header request to `plain/text`.
```
Comment (gsmet) [inline docs/src/main/asciidoc/micrometer.adoc:404]:
```suggestion
By default, the  metrics are exported using the Micrometer format `application/openmetrics-text`,
you can revert to the former format by specifying the `Accept` request header to `plain/text`.
```

**Review by gsmet — CHANGES_REQUESTED** [review]
I suggested some minor improvements.

File: docs/src/main/asciidoc/telemetry-micrometer-tutorial.adoc (hunk 1 above)
Comment (jtama) [reply docs/src/main/asciidoc/telemetry-micrometer-tutorial.adoc]: Used your suggestions and added a curl exemple

File: docs/src/main/asciidoc/micrometer.adoc (hunk 2 above)
Comment (jtama) [reply docs/src/main/asciidoc/micrometer.adoc]: Used your suggestions and added a curl exemple

Comment (ebullient) [reply docs/src/main/asciidoc/micrometer.adoc]: hmm. These are not micrometer formats.. they are defined by Prometheus. You're choosing between Prometheus-defined formats (Micrometer itself has nothing to do with it). Does that matter?

File: docs/src/main/asciidoc/telemetry-micrometer-tutorial.adoc (hunk 1 above)
Comment (jtama) [reply docs/src/main/asciidoc/telemetry-micrometer-tutorial.adoc]: Amended accordingly to @ebullient hint

File: docs/src/main/asciidoc/micrometer.adoc (hunk 2 above)
Comment (jtama) [reply docs/src/main/asciidoc/micrometer.adoc]: Amended accordingly to @ebullient hint

**Review by ebullient — APPROVED** [review]

**Review by gsmet — APPROVED** [review]

**Pull request merged**